lib/active_model/serializable_resource.rb

```diff
@@ -2,7 +2,7 @@
 require 'active_model_serializers/adapter'
 module ActiveModel
   class SerializableResource
-    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter, :meta, :meta_key, :links])
+    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter, :meta, :meta_key, :links, :serialization_context])
```

Comment:
I suppose the problem here will be if a serializer can use url helper to make links without the serialization context.. but that's not anything anyone is working on now